pkg/apis/crd/v1beta1/types.go

```diff
@@ -859,6 +859,15 @@ type EgressSpec struct {
 	// same index in EgressIPs and ExternalIPPools are correlated.
 	// Cannot be set with ExternalIPPool.
 	ExternalIPPools []string `json:"externalIPPools,omitempty"`
+	// Bandwidth specifies the rate limit of north-south egress traffic of this Egress.
+	Bandwidth *Bandwidth `json:"bandwidth,omitempty"`
+}
+
+type Bandwidth struct {
+	// Rate specifies the maximum traffic rate. e.g. 300k, 10M
+	Rate string `json:"rate"`
+	// Burst specifies maximum burst for throttle. e.g. 300k, 10M
```

Comment:
same comment as  in the doc, not sure what "maximum burst for throttle" means?

Reply:
Changed.